### PR TITLE
Add pricing card

### PR DESCRIPTION
**Pull request by borbier**
- Add simple pricing card component

### CODE REVIEW

File: src/docs/components/card/pricing/PricingCard.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      </CardActions>
+    </Card>
+  );
+};
+
+// hide-start
+PricingCard.metadata = {
+  title: 'Simple Pricing',
+  description: 'For showing information about pricing packages',
+  path: 'card/pricing',
```
Comment (siriwatknp) [inline src/docs/components/card/pricing/PricingCard.js]: First of all, thank you for your 1st contribution!.

As you can see, card background and frame background is the same which is hard of see.
image

It is my fault that I don't explain in contributing.md but for now
you can control frame background by adding this code after `description` key.
```
  description: "...",
  frameProps: {
    bgcolor: "#f5f5f5",
  },
```

**Comment by siriwatknp** [comment]
@borbier Moreover, can you capture the design & paste it here?

**Comment by siriwatknp** [comment]
#160

**Comment by borbier** [comment]
I've pushed the design in this issue
https://github.com/siriwatknp/mui-treasury/issues/156

**Pull request merged**